src/core/public/core_system.ts

```diff
@@ -44,6 +46,14 @@ export class CoreSystem {
       injectedMetadata,
     });
 
+    this.fatalErrors = new FatalErrorsService({
+      rootDomElement,
+      injectedMetadata: this.injectedMetadata,
+      stopCoreSystem: () => {
```

Comment:
nit: What do you think about a name like `onFirstError` instead? The problem I have with `stopCoreSystem` is that it's named after what it's expected to do rather than the circumstance when it is used. With the current name, when I see something like line 70, it's not clear to me that that action will trigger the core system stop. I can only find that out if I look into the implementation of the FatalErrorsService to see what series of things fire off as a result.